src/Evolution/Systems/GeneralizedHarmonic/BoundaryConditions/BjorhusImpl.hpp

```diff
@@ -0,0 +1,48 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Utilities/ContainerHelpers.hpp"
+
+/// \cond
+class DataVector;
+namespace gsl {
+template <class T>
+class not_null;
+}  // namespace gsl
+/// \endcond
+
+namespace GeneralizedHarmonic::BoundaryConditions {
+namespace detail {
+// @{
+/*!
+ * \brief Computes the expression needed to set bnudary conditions on Dt<VPsi>
```

Comment:
`bnudary`->`boundary`, and I think the `Dt<VPsi>` would be clearer if it was either be explicitly the tag corresponding to the quantity being set or a prose description of the quantity, like 'the time derivative of the characteristic field' or similar.

Reply:
done